src/components/ec/cuda/ec_cuda_resources.c

```diff
@@ -180,7 +180,16 @@ void ucc_ec_cuda_resources_cleanup(ucc_ec_cuda_resources_t *resources)
 {
     int i;
     CUcontext tmp_context;
-
+#if CUDA_VERSION >= 12000
+    CUresult status;
+    unsigned long long int cu_ctx_id;
+
+    status = cuCtxGetId(resources->cu_ctx, &cu_ctx_id);
+    if (ucc_unlikely(status != CUDA_SUCCESS)) {
```

Comment:
is `ucc_unlikely` relevant here ? If I understand correctly this is control path right ?

Reply:
not really relevant, will remove it